src/Pagination/index.jsx

```diff
@@ -264,7 +265,8 @@ class Pagination extends React.Component {
       buttonLabels, pageCount, icons, variant, size,
     } = this.props;
     const { currentPage } = this.state;
-    const isLastPage = currentPage === pageCount;
+    const isLastPage = (currentPage === pageCount);
+    const isDisabled = isLastPage || (pageCount <= 1);
```

Comment:
Did we want to make a similar change to introduce a `isDisabled` variable for the "previous" button as well?